crates/node/Cargo.toml

```diff
@@ -46,6 +46,7 @@ near-time = { workspace = true }
 node-types = { workspace = true }
 prometheus = { workspace = true }
 rand = { workspace = true }
+# rand_core = { workspace = true }
```

Comment:
is there a reason to keep this comment?

Reply:
leftover that needs to be removed, at the end I managed to achieve what I needed without it